alex's tasks

```diff
@@ -0,0 +1,59 @@
+1 Run plain ruby code
+with irb
+with ruby command + .rb file
+use require and require_relative to load code from another file
+
+2 Know where the official ruby documentation is and actually use it
+ruby-doc.org
+
+3 Built-in types
+Strings
+Symbols
+Arrays and Hashes
```

Comment:
Arrays - an object that stores other objects
words = ["one", "two", "three"] #variable to store our array
words << "four" #add element to end of array using shovel operator 
puts words[3]

Reply:
Hashes - assign values to keys so they can be looked up